miio/gateway/sub_device/lock.py

```diff
@@ -0,0 +1,39 @@
+"""Xiaomi Zigbee Locks."""
+
+from .sub_device import SubDevice
+
+
+class DoorLockS1(SubDevice):
```

Comment:
@rytilahti true, I will make a base class.
I did not do that yet because we have now idea how to control locks, so I don't have any code to put in the base class.
But I will make the base classes anyway for the future.

Reply:
done